css/new.css

```diff
@@ -0,0 +1,137 @@
+body{
+	background-color: #f9f9f9;
+}
+
+.main-section{
+	width: 80%;
+	margin: 0 auto;
+	max-width: 1200px;
+}
+
+form{
+	margin-bottom: 20px;
+	display: inline-block;
+}
+
+.form-section-first{
+    display: flex;
+	justify-content: space-around;
+}
+
+.form-section-first input{
+	width: 31%;
+	border: 0;
+	padding-left: 10px;
+	height: 40px;
+	color: lightgrey;
+	font-size: 20px;
+	padding-right: 10px;
+}
+
+.form-section-first input::placeholder{
+	
+	border-bottom: 1px solid lightgrey;
+	color: lightgrey;
+
+}
+
+.form-section-first input:focus{
+	outline:none;
+}
+
+.form-section span{
+	font-size: 25px;
+	margin-left: 30px;
+	margin-bottom: 30px;
+}
+
+header{
+	margin-bottom: 100px;
+}
+
+fieldset, .form-section, .form-section-first{
+	margin-bottom: 30px;
+}
+
+table{
+	width: 100%;
+	border-collapse: collapse;
+	text-align: center;
+	margin-top: 10px;
+}
+
+textarea{
+	resize: none;
+	height: 100px;
+	border: 2px solid darkgrey;
+	box-sizing: border-box;
+	padding: 10px;
+	margin-top: 10px;
+	font-size: 15px;
+	width:100%;
+}
+
+.firstrow {
+text-align: center;	
```

Comment:
We really need to include a code formatter into our development flow. I, for one, tend to waste more time than I'd like to admit on code that's not properly and consistently formatted. I blame it on my OCD...